scripts/linters/js_ts_linter.py

```diff
@@ -284,6 +284,54 @@ def _check_any_type(self):
 
         return [summary_message]
 
+    def _check_http_requests(self):
+        """Checks if the http requests are made only by
+        backend-api.service.ts.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting HTTP requests check')
+            python_utils.PRINT('----------------------------------------')
+
+        http_client_pattern = r':\n? *HttpClient'
+
+        excluded_files = [
+            'core/templates/services/request-interceptor.service.spec.ts'
+        ]
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+
+            for file_path in self.all_filepaths:
+                if file_path in excluded_files:
+                    continue
+
+                file_content = FILE_CACHE.read(file_path)
+
+                if re.findall(http_client_pattern, file_content):
+                    if not file_path.endswith('backend-api.service.ts'):
+                        failed = True
+                        python_utils.PRINT(
+                            '%s --> An instance of HttpClient is found in this '
+                            'file. You are not allowed to create http requests '
+                            'from files that are not backend api services.' % (
+                                file_path))
+                        python_utils.PRINT('')
```

Comment:
```suggestion
                        summary_message = (
                            '%s --> An instance of HttpClient is found in this '
                            'file. You are not allowed to create http requests '
                            'from files that are not backend api services.' % (
                                file_path))
                        summary_messages.append(summary_message)
                        python_utils.PRINT(summary_message)
                        python_utils.PRINT('')
```

We are changing the linter message format so change it to the suggestion.

Reply:
done